Source/Menu/MainForm.cs

```diff
@@ -264,6 +262,31 @@ orderby tool.Text
             }
         }
 
+        private void LoadDocuments(List<ToolStripItem> docs, string folderPath, string code)
+        {
+            foreach (var filePath in Directory.GetFiles(folderPath))
+            {
+                var ext = System.IO.Path.GetExtension(filePath);
+                var name = System.IO.Path.GetFileNameWithoutExtension(filePath);
+                // These are the following formats that can be selected.
+                if (".pdf .doc .docx .pptx .txt".Contains(ext))
```

Comment:
```suggestion
                if (new[] { ".pdf", ".doc", ".docx", ".pptx", ".txt" }.Contains(ext.ToLowerInvariant()))
```
You need to test for the presence of the entire extension here; your current check passes partial matches such as ".p"  (.pdf) and ".tx" (.txt).

Reply:
Good catch; definitely a bug.